### ISSUE TITLE
May i Know which Dataset do you use for training for Indian language

### ISSUE THREAD

**Issue by kartikeykushwaha14** [k2-fsa/OmniVoice#101]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

i am training it for sankrit language to improve accuracy but it forgot some previous language so can you provide solution so it not forgot previous language and if you provide some dataset name which you used it be helpfull for me 

and if i can solve this by chaging in config you can also tell me that 

### Steps to Reproduce

train on sanskrit 30000s 40 hour data 

### ✔️ Expected Behavior

it should be good in sanskrit and previous language 

### ❌ Actual Behavior

it forgot previous language and sometime generate diffrent language but it follow sanskrit accent

**Comment by zhu-han** [comment]
If you fine-tune on a new dataset, knowledge forgetting is expected to occur. You can try using a smaller learning rate and fewer training updates to alleviate this issue.

The complete list of training data appears in the appendix of our paper (see “A Complete List of Multilingual Training Data”).

**Comment by kartikeykushwaha14** [comment]
Hi,

Thank you for your response — it was very helpful.

Thanks again for your support.

Also, thank you for this great project and the excellent work you’ve done!

Best regards.
